nemo_skills/pipeline/generate.py

```diff
@@ -44,109 +44,231 @@
 # TODO: add num_jobs here for consistency with eval?
 
 
-def _create_commandgroup_from_config(
-    generation_cmd: str,
-    server_config: Optional[Dict],
-    with_sandbox: bool,
-    sandbox_port: Optional[int],
+def _normalize_models_config(
```

Comment:
maybe move both new funcs to a utils file as this will be shared across pipelines

Reply:
sure--do you see any reason why moving them into the `nemo_skills.pipeline.utils.generation` file would not work?

Reply:
no reason, should be good to move there

Reply:
updated in da2fdd6